en/documentation/index.md

```diff
@@ -12,9 +12,15 @@ Guides, tutorials, and reference material to help you learn more about Ruby
 Although you can easily [try Ruby in your browser][1], you can also read
 the [installation guide](installation/) for help on installing Ruby.
 
-### Ruby Language Documentation
+### Official Ruby Documentation
 
-The authoritative Ruby language documentation at [docs.ruby-lang.org][docs-rlo-en] is the place to start if you want to understand Ruby. It covers all aspects of the language and should be the first place you look when learning or referencing Ruby programming language. Other resources listed below are complementary to the official language documentation or they present Ruby documentation in a different format/style/flavor.
+[docs.ruby-lang.org/en][docs-rlo]: List of documentation for all Ruby versions released after 2.1.
+
+[docs.ruby-lang.org/en/3.4][docs-rlo-3.4]: Documentation for Ruby 3.4.
+
+[docs.ruby-lang.org/en/master][docs-rlo-master]: Documentation for the latest Ruby version (master).
```

Comment:
```suggestion
[docs.ruby-lang.org/en/master][docs-rlo-master]: Documentation for Ruby's master branch.
```

3.4 is the latest Ruby version.  We could use different language here if you prefer, but `latest Ruby version` isn't accurate.